mono/mini/debugger-agent.c

```diff
@@ -4987,6 +4989,77 @@ ss_update (SingleStepReq *req, MonoJitInfo *ji, SeqPoint *sp, DebuggerTlsData *t
 	return hit;
 }
 
+#if IL2CPP_MONO_DEBUGGER
+/*
+* ss_update_il2cpp:
+*
+* Return FALSE if single stepping needs to continue.
+*/
+static gboolean
+ss_update_il2cpp(SingleStepReq *req, DebuggerTlsData *tls, MonoContext *ctx, Il2CppSequencePointC *sequencePoint)
+{
+	gboolean hit = TRUE;
+
+	if (req->async_stepout_method == *sequencePoint->method)
```

Comment:
Do we need to call `il2cpp_mono_methods_match` here instead of using a direct pointer comparison?

Reply:
Probably?  I don't think the async stepout functionality works at the moment, though.

Reply:
If it works with `il2cpp_mono_methods_match` for the passing tests we have now, I would prefer that, as I think it is a "safer" alternative. Then when we get async steopout working, this should not be an issue.